k8s/web.yaml

```diff
@@ -30,7 +30,7 @@ spec:
           resources: {}
           env:
             - name: BASE_URL
-              value: "https://robust.gcloud.netnod.se" 
+              value: "https://dev.robust.gcloud.netnod.se"
```

Comment:
This is why the loading page wasn't working yesterday but was working this morning, and it stopped working after #66 brought back the "bug".

![Screenshot 2022-01-27 at 13 44 21](https://user-images.githubusercontent.com/646665/151361297-27f542af-8493-43ed-bf54-e80ee275c25a.png)